Documentation/gettingstarted/k8s-install-gke-validate.rst

```diff
@@ -0,0 +1,50 @@
+Validate the Installation
+=========================
+
+You can monitor as Cilium and all required components are being installed:
+
+.. parsed-literal::
+
+    kubectl -n cilium get pods --watch
+    NAME                                    READY   STATUS              RESTARTS   AGE
+    cilium-operator-cb4578bc5-q52qk         0/1     Pending             0          8s
+    cilium-s8w5m                            0/1     PodInitializing     0          7s
+    coredns-86c58d9df4-4g7dd                0/1     ContainerCreating   0          8m57s
+    coredns-86c58d9df4-4l6b2                0/1     ContainerCreating   0          8m57s
+
+It may take a couple of minutes for all components to come up:
+
+.. parsed-literal::
+
+    cilium-operator-cb4578bc5-q52qk         1/1     Running   0          4m13s
+    cilium-s8w5m                            1/1     Running   0          4m12s
+    coredns-86c58d9df4-4g7dd                1/1     Running   0          13m
+    coredns-86c58d9df4-4l6b2                1/1     Running   0          13m
+
+Deploy the connectivity test
```

Comment:
The connectivity test is the same for the three versions `Documentation/gettingstarted/k8s-install*validate.rst` (generic, Azure, GKE). Is it worth to extract it and to include it in each of those three files?

Whether we split the file or not, PR looks good to me.

Reply:
Agreed. I'll extract.